src/constants.js

```diff
@@ -17,3 +18,6 @@ export const OTHER_MOTS = [
 export const VALID_MOTS = [...DEFAULT_MOTS, ...OTHER_MOTS];
 
 export const GRAPHHOPPER_MOTS = ['foot', 'car'];
+
+// Currently no 'coach' mot available for stop finder.
+export const handleStopFinderMot = mot => (mot === 'coach' ? 'bus' : mot);
```

Comment:
constants is meant to use for simple objetc , not really  functions.  Can you leave it in routing Menu ?

Reply:
It is a temporary code until coach is available in stopFinder. I tought it would be easier to find this line a code and remove in where the mots are defined.
But I can set in the Menu if you prefer :+1: